SCClassLibrary/Common/Control/Server.sc

```diff
@@ -38,8 +40,6 @@ ServerOptions {
 	var <>threads = nil; // for supernova
 	var <>useSystemClock = false;  // for supernova
 
-	var <numPrivateAudioBusChannels=112;
```

Comment:
Intrinsic classes have their instance/class variables hardcoded on the C++ side. That's definitely true of things like Finalizer and the classes in Kernel. It doesn't seem to be the case here.